engine/src/main/java/org/terasology/rendering/opengl/OpenGLUtils.java

```diff
@@ -61,66 +70,6 @@ public static void applyBillboardOrientation() {
     }
 
     /**
-     * Sets the viewport of the currently bound FBO to the dimensions of the FBO
```

Comment:
The following methods have been converted to StateChanges and are unused here, so removing to eliminate confusion and code redundancy.